src/components/scenes/CreateWalletSelectCryptoScene.tsx

```diff
@@ -58,7 +58,7 @@ const CreateWalletSelectCryptoComponent = (props: Props) => {
       return map
     }, {})
   )
-  const [numSelected, setNumSelected] = React.useState(0)
+  const [numSelected, setNumSelected] = React.useState(Object.values(selectedItems).filter(Boolean).length)
```

Comment:
Interesting use of `Boolean`. I would have done `item => item != null` or whatever explicit check was appropriate, but ⭐ for creative shortcuts!